src/reconstruct_loopset.jl

```diff
@@ -239,11 +239,12 @@ function calcnops(ls::LoopSet, os::OperationStruct)
     offsets = ls.loopsymbol_offsets
     idxs = loopindex(ls, os.loopdeps, 0x04)  # FIXME DRY
     iszero(length(idxs)) && return 1
-    return map(i->offsets[i+1]-offsets[i], idxs)
+    return maximum(i->offsets[i+1]-offsets[i], idxs)
 end
```

Comment:
For now, only loads and stores are allowed to have a mix of `nops` as inputs.
`nops` for loads and stores are also always `1`, so we don't need a vector.